ckanext/issues/lib/helpers.py

```diff
@@ -95,9 +95,10 @@ def issues_enabled(dataset):
             return True
     else:
         extras = dataset.get('extras')
```

Comment:
Wish I saw your PR earlier, would have saved me some time. You can fix this a bit more concisely and with less nesting: https://github.com/deniszgonjanin/ckanext-issues/commit/47e2aa0c3c0695f314daa4243b9c2219d63b63f1